morpho-checkpoint-node/tests/Test/Morpho/Generators.hs

```diff
@@ -222,3 +225,40 @@ testBftConfig =
       bftSignKey = SignKeyMockDSIGN 1,
       bftVerKeys = mempty -- not used to forge blocks
     }
+
+-- Copied from https://github.com/haskell/aeson/blob/d711df76b826942f4a9e791712512c6b19b8c1c8/tests/Instances.hs#L170-L203
```

Comment:
Copied that code here because it wasn't published yet by Aeson?

Reply:
It's part of their test suite, so it's unlikely that it even will be published. Duplicating it seems to be the easiest way for now.